docs/guidelines/styles.rst

```diff
@@ -32,6 +137,7 @@ Use block-style and group elements underneath:
 #. border, background (box style data)
 #. media, print (media queries)
 #. :after, :before, :active (pseudo elements)
+#. nested elements or parent referencing selectors
 
 Combine attributes such as background-image, background-color, background-repeat into
```

Comment:
why is this a thing?
usually problems arise when you want to override let's say only background-image and you can't, because shorthand trumps everything